helpme/templates/helpme/ticket_detail.html

```diff
@@ -58,7 +58,7 @@ <h4><small>Status:</small> <span class="badge badge-pill badge-primary">{{ ticke
 
     <hr>
     <h4>Add Comment</h4>
-    <form method="post" action="{% url 'helpme-api:create-comment' ticket.uuid %}" class="text-center">
+    <form method="post" action="{% url 'create-comment' ticket.uuid %}" class="text-center">
```

Comment:
I'd recommend naming it "helpme-api-create-comment" to reduce the chances of a collision.